src/ControlSystem/Controller.hpp

```diff
@@ -8,6 +8,12 @@
 
 #include "DataStructures/DataVector.hpp"
 
+/// \cond
+namespace PUP {
+class er;
+}
```

Comment:
Closing namespace comment.

Reply:
I thought if it was inside `\cond ... \endcond` I didn't need it?

Reply:
`\cond` is just a doxygen thing.